data/helpers.d/getopts

```diff
@@ -98,10 +98,10 @@ ynh_handle_getopts_args () {
 
 				if [ "$parameter" = "?" ]
 				then
-					ynh_die "Invalid argument: -${OPTARG:-}"
+					ynh_die --message="Invalid argument: -${OPTARG:-}"
```

Comment:
Uuuh, is the --message already implemented ? :D

Reply:
Indeed !
I just picked up the code from the other PR !